common/gtk-3.0/meson.build

```diff
@@ -42,8 +42,10 @@ gtk3_asset_names = run_command(
 
 assets_svg = gtk3_ver / 'assets.svg'
 
+gtk3_assets = []
+gtk3_hidpi_assets = []
 foreach asset : gtk3_asset_names
```

Comment:
Can you add a empty line between the initial array declarations, and the foreach loop.

Otherwise this LGTM! :)